src/components/Dropdown.vue

```diff
@@ -22,22 +26,45 @@ export default {
       optionsVal: this.options,
     };
   },
+  computed: {
+    displayValue() {
+      const selectedOption = this.optionsVal.find((option) => option.value === this.selectedVal);
+      return selectedOption ? selectedOption.text : '';
+    },
+  },
 };
 </script>
 
 <style lang="scss" scoped>
-.aggregator__dropdown{
+.dropdown {
```

Comment:
can you replace it with this (few tweaks, but cant make suggestion because github limitations):
```
.dropdown {
  display: inline-block;
  position: relative;
  min-width: 5rem;

  .display {
    text-align: center;
    position: relative;
    color: $standard_font_color;
    font-size: .75rem;
    background-color: $light_color;
    padding: .6rem 2rem .7rem .85rem;
    border-radius: .25rem;
    line-height: .9rem;
    min-height: 2.2rem;

    img {
      @include vertical-align(absolute);
      right: .85rem;
    }

    &:hover {
      cursor: pointer;
    }
  }

  select {
    bottom: 0;
    left: 0;
    opacity: 0;
    position: absolute;
    right: 0;
    top: 0;
  }
}